### PR TITLE
Added French translation

### PR DESCRIPTION
**Pull request by Yenaman**

### CODE REVIEW

File: fr/commands.po [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+msgstr "t"
+
+msgid "translate"
+msgstr "traduire"
+
+msgid "translator"
+msgstr "traducteur"
+
+msgid "about"
+msgstr "à propos"
```
Comment (aziznatour) [inline fr/commands.po]: can this be a single word?

File: fr/commands.po [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+msgstr "avantages"
+
+msgid "custom"
+msgstr "customisation"
+
+msgid "last"
+msgstr "dernier"
+
+msgid "detect"
+msgstr "détecter"
```
Comment (aziznatour) [inline fr/commands.po:70]: do users usually write the accents? keep in mind this is a command string, needs to be written easily, and keep in mind people on mobile device

File: fr/errors.po [hunk 3]
```diff
@@ -0,0 +1,392 @@
...
+msgstr ""
+"Pour traduire un message, vous *avez besoin* d'un message !\n"
+"Merci d'éditer votre commande pour y inclure du texte."
+
+msgid "Invalid languages argument"
+msgstr "Arguments de langue invalides"
+
+msgid "Incorrect format for from/to argument, please edit to fix."
+msgstr ""
+"Format incorrect de l'argument from/to, merci d'éditer pour corriger le problème."
```
Comment (aziznatour) [inline fr/errors.po]: from/to not translated?

File: fr/languages.po [hunk 4]
```diff
@@ -0,0 +1,324 @@
...
+msgstr "Roumain"
+
+msgid "Russian"
+msgstr "Russe"
+
+msgid "Samoan"
+msgstr "Samoan"
+
+msgid "Gaelic Scottish"
+msgstr "Gaélique (Écosse)"
```
Comment (aziznatour) [inline fr/languages.po]: possible to remove the `()` ?

File: fr/main.po [hunk 5]
```diff
@@ -0,0 +1,876 @@
...
+
+msgid "Will not work in **ignored / blocked** channels"
+msgstr "Ne marche pas dans les salons **ignorés / bloqués**"
+
+msgid "Will not work if user has %s role"
+msgstr "Ne marche pas si l'utilisateur a le rôle %s"
+
+msgctxt "example"
+msgid "Hello World"
+msgstr "Hello World"
```
Comment (aziznatour) [inline fr/main.po]: This needs to be translated 😅

**Review by aziznatour — CHANGES_REQUESTED** [review]
Thank you so much! Please review the comments for improvement :)

File: fr/commands.po (hunk 2 above)
Comment (Yenaman) [reply fr/commands.po:70]: Accents in French are a big thing

File: fr/commands.po (hunk 1 above)
Comment (Yenaman) [reply fr/commands.po]: Could be "Informations"

Comment (aziznatour) [reply fr/commands.po]: Acceptable

File: fr/errors.po (hunk 3 above)
Comment (Yenaman) [reply fr/errors.po]: Hmm we thought this was an command argument, can we have some context?

File: fr/commands.po (hunk 2 above)
Comment (aziznatour) [reply fr/commands.po:70]: very well then, keep them 😅

**Pull request merged**